packages/core/rendering/core-rendering-server-internal/src/rendering_service.tsx

```diff
@@ -201,14 +205,17 @@ export class RenderingService {
 
     const loggingConfig = await getBrowserLoggingConfig(this.coreContext.configService);
 
+    const translationHash = i18n.getTranslationHash();
+    const translationsUrl = `${serverBasePath}/translations/${translationHash}/${i18nLib.getLocale()}.json`;
+
```

Comment:
Changing the `translationsUrl` that will be sent to the browser to load the translations